liblocation/src/main/java/com/mapbox/services/android/core/location/LocationEnginePriority.java

```diff
@@ -11,4 +16,9 @@
   public static final int LOW_POWER = 1;
   public static final int BALANCED_POWER_ACCURACY = 2;
   public static final int HIGH_ACCURACY = 3;
+
+  @IntDef({NO_POWER, LOW_POWER, BALANCED_POWER_ACCURACY, HIGH_ACCURACY})
+  @Retention(RetentionPolicy.SOURCE)
+  public @interface PowerMode {
```

Comment:
We should use `@LocationEnginePriority.PowerMode` instead of the previous constants across `liblocation` module classes.